packit/distgit.py

```diff
@@ -82,6 +88,17 @@ def local_project(self):
             )
         return self._local_project
 
+    @property
+    def downstream_config(self) -> Optional[PackageConfig]:
+        if not self._downstream_config:
+            try:
+                self._downstream_config = get_local_package_config(
+                    self.local_project.working_dir
+                )
+            except PackitConfigException:
+                return None
```

Comment:
this looks pretty odd

Reply:
We do not need to fail if the config does not exist...

Reply:
That's the reason I originally returned `None` also in the `get_local_package_config`....